calibr8/tests.py

```diff
@@ -941,6 +1043,73 @@ def test_likelihood(self):
         assert x_dense[numpy.argmax(actual)] == 2
         pass
 
+    @pytest.mark.xfail(reason="Draft test case, see https://github.com/JuBiotech/calibr8/issues/15")
+    def test_likelihood_nobroadcasting_fallback(self):
+        class _TestSwitchableBroadcastingModel(calibr8.CalibrationModel, calibr8.NormalNoise):
+            def __init__(self):
+                self.x_shapes = []
+                self.can_broadcast = False
```

Comment:
For the test I needed a model that incorrectly does `.sum()`, thereby breaking any broadcasting. Otherwise the else branch in the likelihood is not covered.
Switching like this seemed easier than implementing and instatiating two test models.